### ISSUE TITLE
Implement LWdb postWord

### ISSUE THREAD

**Issue by hhzl** [hhzl/LearnWords2#36]
cf. https://pouchdb.com/api.html#create_document

db.post()

**Comment by hhzl** [comment]
Create a new document and let LWdb  auto-generate an _id for it.

**Comment by phyiction** [comment]
I thought I implemented this in #43.

**Comment by hhzl** [comment]
Yes but the implementation of `db.put(aWord)` is incomplete

https://github.com/hhzl/LearnWords2/blob/master/src/LWdb.js#L85

1. I do not understand why we need a clone of `aWord`. Because later we only want the string version of it. 
2. We need to check if `aWord` already has `_id`. If yes use that one. If no generate an `_id`
3. If a new word was added we need to call `db.invalidateIndex()`

**Comment by phyiction** [comment]
I agree we do not need to clone `aWord`. Every time we retrieve a word we're getting it from localStorage. What happens when I put the same word into the database?

**Comment by hhzl** [comment]
I have updated the [API description](https://github.com/hhzl/LearnWords2/blob/master/docs/Developer_notes.md#lw2-master-branch).

We still need to describe the semantics of db.put(aWord). It is incomplete.

For the db.put question I opened https://github.com/hhzl/LearnWords2/wiki

**Comment by hhzl** [comment]
Moved to release 0.3. We do not need this for a MVP implementation of the libary in release 0.2
